tests/OpenTelemetry/Integration/API/BaggageTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace DDTrace\Tests\OpenTelemetry\Integration\API;
+
+use DDTrace\Tag;
+use DDTrace\Tests\Common\BaseTestCase;
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Common\SpanAssertionTrait;
+use DDTrace\Tests\Common\TracerTestTrait;
+use OpenTelemetry\API\Baggage\Baggage;
+use OpenTelemetry\API\Baggage\Propagation\BaggagePropagator;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+
+final class BaggageTest extends BaseTestCase
```

Comment:
we support Baggage in PHP?

Reply:
The [Baggage](https://github.com/open-telemetry/opentelemetry-php/blob/main/src/API/Baggage/Baggage.php#L12) API is entirely dependent on the Context API, which we support. The question is, do we fully support it? To that, I would answer that I haven't been able to determine how we wouldn't support it, whether it is in these tests or in an app with `OTEL_PROPAGATORS=baggage,tracecontext`  (which led to the trace in the PR Description).

Reply:
yup